action.yml

```diff
@@ -0,0 +1,13 @@
+name: 'Office 365 CLI Run Script'
+description: 'Runs an Office 365 CLI script'
+inputs:
+  O365_CLI_SCRIPT_PATH:  
+    description: 'Relative path of the script in your repo.'
+  O365_CLI_SCRIPT:
+    description: 'Office 365 CLI Script'
+runs:
+  using: 'docker'
+  image: 'Dockerfile'
+branding:
+  icon: 'chevron-right'  
```

Comment:
@anoopt , do you know if there is a way to plug in our own icon here? It seems you are using built-in icon.

Reply:
@VelinGeorgiev Thank you for letting me know about `terminal` icon. As per the documentation [here](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/metadata-syntax-for-github-actions#branding), we can only use the icons that are part of Feather icons and `terminal` icon is one of them. So I will update the PR accordingly.